common/src/java/com/zimbra/common/soap/AdminConstants.java

```diff
@@ -152,6 +152,13 @@
     public static final String E_GET_ALL_CONFIG_REQUEST = "GetAllConfigRequest";
     public static final String E_GET_ALL_CONFIG_RESPONSE = "GetAllConfigResponse";
 
+    public static final String E_ABQ_CONFIG_REQUEST = "ABQConfigRequest";
+    public static final String E_ABQ_CONFIG_RESPONSE = "ABQConfigResponse";
+    public static final String E_ABQ_CONFIG_NAME = "name";
```

Comment:
There might already be constants for this

Reply:
updates to use existing constants, removed newly added